sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuAggregateExec.scala

```diff
@@ -219,9 +219,6 @@ object AggregateUtils extends Logging {
   ): Boolean = {
 
     var repartitionHappened = false
-    if (hashSeed > 200) {
```

Comment:
Could we please decouple the hashSeed for the number of re-partition times being done? I think the code would be much more readable if we could say how many times we tried to re-partition the data instead of "too many times".